backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/75-de.metas.fresh.reports/5617251_sys_gh12114_Add_IsPrintWhenPackingMaterial_in_Docs_Sales_InOut_Details_HU.sql

```diff
@@ -0,0 +1,55 @@
+
+DROP FUNCTION IF EXISTS de_metas_endcustomer_fresh_reports.Docs_Sales_InOut_Details_HU ( IN Record_ID numeric, IN AD_Language Character Varying (6) );
+DROP TABLE IF EXISTS de_metas_endcustomer_fresh_reports.Docs_Sales_InOut_Details_HU;
+
+CREATE TABLE de_metas_endcustomer_fresh_reports.Docs_Sales_InOut_Details_HU
+(
+	MovementQty numeric,
+	Name Character Varying,
+	UOMSymbol Character Varying (10),
+	Description Character Varying,
+	IsPrintWhenPackingMaterial char(1)
+);
+
+
+CREATE FUNCTION de_metas_endcustomer_fresh_reports.Docs_Sales_InOut_Details_HU ( IN Record_ID numeric, IN AD_Language Character Varying (6) )
+RETURNS SETOF de_metas_endcustomer_fresh_reports.Docs_Sales_InOut_Details_HU AS
+$$
+SELECT
+	SUM(iol.QtyEntered)			AS MovementQty,
+	COALESCE(pt.Name, p.name)		AS Name,
+	COALESCE(uomt.UOMSymbol, uom.UOMSymbol)	AS UOMSymbol,
+	iol.Description,
+	p.IsPrintWhenPackingMaterial
+FROM
+	M_InOut io
+	INNER JOIN M_InOutLine iol 			ON io.M_InOut_ID = iol.M_InOut_ID AND iol.isActive = 'Y'
+	-- Product and its translation
+	LEFT OUTER JOIN M_Product p 			ON iol.M_Product_ID = p.M_Product_ID AND p.isActive = 'Y'
+	LEFT OUTER JOIN M_Product_Trl pt 		ON iol.M_Product_ID = pt.M_Product_ID AND pt.AD_Language = $2 AND pt.isActive = 'Y'
```

Comment:
please prefix the parameter names with `p_` and use them in the function body, intead of `$1`, `$2` etc